x-pack/plugins/apm/server/lib/service_map/get_service_anomalies.ts

```diff
@@ -104,7 +104,7 @@ export async function getServiceAnomalies({
     },
   };
 
-  const response = await ml.mlSystem.mlAnomalySearch(params);
+  const response = await ml.mlSystem.mlAnomalySearch(params, mlJobIds);
```

Comment:
Does this mean that the filter by `mlJobIds` is now redundant? https://github.com/elastic/kibana/blob/3a9782d996245c6db0b3277e8b982598431ad64a/x-pack/plugins/apm/server/lib/service_map/get_service_anomalies.ts#L64

Reply:
no this will still be needed in the query.
We need an explicit list of job ids passed to the function so we can check they are available in the current space.
We could attempt to retrieve them from the query, but each query object could be different and the job filtering done in a different way.